### PR TITLE
Cleaned up gems

### PR DESCRIPTION
**Pull request by JoshBroomberg**
- Dropped a total of 27 gems
- Added versions to all gems

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,37 +5,22 @@ ruby "2.3.1"
...
-gem "passenger" # Web server.
-gem "resque", "~> 1.22.0" # Background job processing.
-gem "flip" # Feature flags.
-gem "squeel"
+gem "pg", "0.18.4"
+gem "passenger", "5.0.29" # Web server.
+gem "squeel", "~> 1.2.3"
 
 # Production Development
-gem "honeybadger", "~> 2.0" # Error reporting.
```
Comment (khanotations) [inline Gemfile]: I think we should keep this

File: Gemfile [hunk 2]
```diff
@@ -5,37 +5,22 @@ ruby "2.3.1"
...
-gem "informant-rails" # Performence monitoring.
 gem "active_record_query_trace", "~> 1.5" # Trace queries to lines of code.
-gem "pry-rails" # Improved production console.
+gem "pry-rails", "~> 0.3.4" # Improved production console.
 
 # External services
 gem "aws-sdk", "< 2.0" # S3 storage.
-gem "omniauth-facebook" # Facebook login.
-gem "redcarpet" # Markdown
-gem "intercom-rails"
```
Comment (khanotations) [inline Gemfile]: let's keep intercom? or does segment replace it

File: Gemfile [hunk 3]
```diff
@@ -5,37 +5,22 @@ ruby "2.3.1"
 # System
 gem "rake", "11.2.2"
 gem "rails", "4.2.2"
-gem "pg"
-gem "passenger" # Web server.
-gem "resque", "~> 1.22.0" # Background job processing.
-gem "flip" # Feature flags.
-gem "squeel"
+gem "pg", "0.18.4"
```
Comment (khanotations) [inline Gemfile]: Let's stick with `~> 0.18.4` to avoid getting super specific. Same as below

File: Gemfile [hunk 4]
```diff
@@ -49,10 +34,6 @@ gem "momentjs-rails", ">= 2.9.0"
 gem "bootstrap3-datetimepicker-rails", "~> 4.17.37"
 gem "rails_12factor", group: :production # For Heroku assets
 
-# Charts
-gem "groupdate"
```
Comment (khanotations) [inline Gemfile]: I'd leave these here but commented out

File: Gemfile (hunk 1 above)
Comment (JoshBroomberg) [inline Gemfile]: We will keep honeybadger, but I think the principal should be that I include the gem when I configure the service.

File: Gemfile (hunk 2 above)
Comment (JoshBroomberg) [inline Gemfile]: Segment #ftw

File: Gemfile (hunk 3 above)
Comment (JoshBroomberg) [inline Gemfile]: cool

File: Gemfile (hunk 4 above)
Comment (JoshBroomberg) [inline Gemfile]: We always have ixstudents as a reference (and the git history). I think we should incude gems when we use them as a rule

**Comment by JoshBroomberg** [comment]
@khanotations good to merge here now

**Pull request merged**